jdbc-datasource/README.adoc

```diff
@@ -0,0 +1,73 @@
+= Connecting to a JDBC DataSource: A Camel Quarkus example
+:cq-example-description: An example that shows how to define a Camel route in XML for tokenizing a CSV a file.
+
+{cq-description}
+
+In particular, it demonstrates the following:
+
+1. Defining a DataSource
+2. Querying the Database defined in the previous DataSource
+4. Usage of properties defined in `application.properties`
+5. No Java code required or used, the route defined in XML can still be compiled to native code.
+
+This example will connect to an H2 database with the connection details defined in `application.properties`.
+If the example is run on Development mode and no database exists, Quarkus will create a matching database
+https://quarkus.io/guides/datasource#dev-services[as described here].
+
+TIP: Check the https://camel.apache.org/camel-quarkus/latest/first-steps.html[Camel Quarkus User guide] for prerequisites
+and other general information.
+
+== Start in the Development mode
+
+[source,shell]
+----
+$ mvn clean compile quarkus:dev
+----
+
+The above command compiles the project, starts the application and lets the Quarkus tooling watch for changes in your
+workspace. Any modifications in your project will automatically take effect in the running application.
+
+TIP: Please refer to the Development mode section of
+https://camel.apache.org/camel-quarkus/latest/first-steps.html#_development_mode[Camel Quarkus User guide] for more details.
+
+Once the application has started it will wait 5 seconds and then read the file every 30 seconds. By default the row split
+is sequential however you can edit the properties file and set the parallel processing to true. The output should then be
```

Comment:
Maybe we need a "," or "." before the _however_.

Reply:
Oh, that's a part I didn't update that comes from the example I used as template. Removing it now.

Reply:
> I guess you'd need to add the example in the https://github.com/apache/camel-quarkus-examples/blob/main/.github/test-categories.yaml, so that it could get compiled by the CI build

Should I create a new category for database things?

Reply:
It's fine to add to an existing one.